econml/dml.py

```diff
@@ -497,8 +544,21 @@ def fit(self, Y, T, X=None, W=None, *, sample_weight=None, sample_var=None, grou
         self
         """
         return super().fit(Y, T, X=X, W=W, sample_weight=sample_weight, sample_var=sample_var, groups=groups,
+                           cache_values=cache_values,
                            inference=inference)
 
+    def refit_final(self, *, inference='auto'):
+        return super().refit_final(inference=inference)
+    refit_final.__doc__ = _OrthoLearner.refit_final.__doc__
+
+    @property
+    def bias_part_of_coef(self):
+        return self.rlearner_model_final._fit_cate_intercept
```

Comment:
Same here, why we need this attribute if it's same as `fit_cate_intercept`  ? Another corner case I think we are missing now is from the `parse_final_model_params`, what if the user sets `fit_cate_intercept=False` but input the final model with `fit_intercept=True`, we should update our `fit_cate_intercept` depends on whether there is an intercept in the user defined final model?

Reply:
fit_cate_intercept is different than the intercept of the final model. The fit_cate_intercept is aobut whether we augment X with an constant of 1's. The fit intercept of the final model is about fitting an extr aoffset in the regresstion not a cate offset.

Reply:
Also observe that bias_part_of_coef, returns the fit_cate_intercept based on the fit time. See the implementation in of bias_part_of_coef in dml. It doesn't read the fit_cate_intercept, but it reads the parameter that was stored when fit was called. 
So there wont be an inconsitency here. At least that was a corner case that I tried to address.